jupyterlab_automl/handlers.py

```diff
@@ -88,19 +88,35 @@ def get_table_specs(client, datasetId):
 
 
 def get_datasets(client, parent):
-    datasets = client.list_datasets(parent)
-    return {
-        "datasets": [
-            {
+    datasets = []
+    for dataset in client.list_datasets(parent):
+        dataset_type = "other"
+        metadata = ""
+        if dataset.tables_dataset_metadata.primary_table_spec_id != "":
```

Comment:
I am kind of hesitant about this method of checking what type of dataset it is because it seems kind of inefficient. When the dataset object come over on the api though it's hard to tell what metadata it has without doing all this empty string checks. take a look at this and let me know what you think